opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp

```diff
@@ -352,7 +352,7 @@ class BrineCo2Pvt
         Scalar M_CO2 = CO2::molarMass();
         Scalar M_H2O = H2O::molarMass();
 
-        const LhsEval& tempC = temperature - 273.15;        /* tempC : temperature in �C */
+        const LhsEval& tempC = temperature - 273.15;        /* tempC : temperature in °C */
```

Comment:
I doesn't show in the GitHub review interface, but I **assume** you've only changed the encoding of the degrees Centigrade symbol here, right?